src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4686,3 +4686,132 @@ def test_ps_s3_notification_push_kafka_security_ssl_sasl_scram():
 def test_ps_s3_notification_push_kafka_security_sasl_scram():
     kafka_security('SASL_PLAINTEXT', mechanism='SCRAM-SHA-256')
 
+
+@attr('data_path_v2_test')
+def test_ps_s3_data_path_v2_migration():
```

Comment:
this is a great test to cover migration when there are persistent notifications in the queue.
can you also add a similar test for the simpler case of non-persistent notification?
- change to v1
- setup topic and notifications
- make sure they work
- change to v1
- make sure they still work